apiary.apib

```diff
@@ -542,7 +542,186 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             errors: "Some error message"
         }
 
+## Mentor Request | Create a New Mentor Request [/api/v1/requests{?user_id,details,language,service_id,requested_mentor_id,status,assigned_mentor_id}]
```

Comment:
Since the model name is `Request` and the endpoint is also `../requests`, can you please all of these endpoints to begin with "Request ....".  For example, "Request a Mentor". 

And then re-alphabetize all of the Request related endpoints accordingly.

Reply:
I'll fix them to be like this: 
`## Mentor Request | Request .... r [/api/vi/....]`

Reply:
They'll need to go by their **model**, `Request`, so it would be:

```
## Request | Request a Mentor .... r [/api/vi/....]
```